plugins/action_remove_user_org/src/removeUserOrg.ts

```diff
@@ -0,0 +1,39 @@
+import { ActionQueueStatus } from '../../../src/generated/graphql'
+import { ActionPluginInput } from '../../types'
+
+const removeUserOrg = async function ({ parameters, DBConnect }: ActionPluginInput) {
+  const { userId, orgId, user_id, organisation_id, org_id } = parameters
+  try {
+    console.log(`\Removing user from organisation...`)
+    const result = await DBConnect.removeUserOrg({
+      userId: userId ?? user_id,
+      orgId: orgId ?? org_id ?? organisation_id,
```

Comment:
Most of our actions now use `camelCase`, so I wanted to keep this pattern. But we accept `snake_case` versions here too, for consistency with `joinUserOrg`

(Shall I update that one to accept both versions too?)

Reply:
Yes, that's a good call. As long as it's adding more options, that's good. And also for consistency. Can we also allow all options as `orgId ?? org_id ?? organisationId ?? organisation_id` in both actions please?

Reply:
Cool, done: e8d7066c

I also added a uniqueness constraint to the user_org table, and handled the "userRole" parameter (which we had documented but not implemented 😳)